sources/config.cpp

```diff
@@ -76,6 +75,7 @@ void Config::setConfig(const QVariantMap &config) {
     m_config = config;
     syncConfigToCache();
     emit configChanged();
+    writeConfig();
```

Comment:
This class has been enhanced in the develop branch.
- Unfortunately setConfig cannot be enhanced with a bool return since it's a Q_PROPERTY
- The client must call isValid() after calling setConfig